code/modules/surgery/organs/ears.dm

```diff
@@ -37,9 +37,6 @@
 	if(HAS_TRAIT_NOT_FROM(owner, TRAIT_DEAF, EAR_DAMAGE))
 		return
 
-	if((damage < maxHealth) && (organ_flags & ORGAN_FAILING)) //ear damage can be repaired from the failing condition
```

Comment:
I've observed a possible bug with ears no longer being restored by inacusiate, there's a chance this change is the culprit.